packages/api/internal/handlers/template_update.go

```diff
@@ -71,46 +63,48 @@ func (a *APIStore) PatchTemplatesTemplateID(c *gin.Context, aliasOrTemplateID ap
 
 	if template.TeamID != team.ID {
 		a.sendAPIStoreError(c, http.StatusForbidden, fmt.Sprintf("You don't have access to sandbox template '%s'", aliasOrTemplateID))
```

Comment:
I would prefer to return the same error message and HTTP code (ideally 404) for "not found" and "found but you don't have access". This will not give a clue to a possible attacker that some IDs exist. 

But it depends how we handled it in different endpoints.
If we are returning 2 different codes then its probably okay.